src/test/java/com/razorpay/InvoiceClientTest.java

```diff
@@ -0,0 +1,140 @@
+package com.razorpay;
+
+import org.json.JSONObject;
+
+import org.junit.Test;
+import org.mockito.InjectMocks;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class InvoiceClientTest extends BaseTest{
+
+    @InjectMocks
+    protected InvoiceClient invoiceClient = new InvoiceClient(TEST_SECRET_KEY);
+
+    private static final String INVOICE_ID = "inv_DAweOiQ7amIUVd";
+
+    @Test
+    public void create() throws RazorpayException {
```

Comment:
choose test name more descriptive , 
Like ShouldReturnValidEntityIfResponseDoesNotHave

Reply:
or may be you can add java docs for more info